src/scripts/qa.ts

```diff
@@ -80,6 +63,22 @@ async function handler(): Promise<void> {
   validateChain(chain);
   validateEnvironment(env);
 
+
+  const evmQaAddress = process.env.QA_TAKER_ADDRESS?.toLowerCase();
+  if (!evmQaAddress) {
+    throw new Error(
+        `Please specify the taker address you want to use for QA in src/.env under QA_TAKER_ADDRESS`
+    );
+  }
+  validateEvmAddress(evmQaAddress);
+  const solanaQaAddress = process.env.SOLANA_QA_TAKER_ADDRESS;
+  if (!solanaQaAddress) {
+    throw new Error(
+        `Please specify the taker address you want to use for QA in src/.env under SOLANA_QA_TAKER_ADDRESS`
+    );
+  }
+  validateSolanaAddress(solanaQaAddress);
```

Comment:
I don't think we should require current EVM-only makers to supply a Solana address. Can we implement it so that the QA will only run for the chains where an address was provided?